File: Maths/1_count_digits.cpp
#include<iostream>
using namespace std;

int countDigit(int n){
    int count=0;
    while(n){
      count++;
      n=n/10;
    }
    return count;
}
int main(){
    int num=12345;
    cout<<"Total Number of Digits in "<<num<<" is "<<countDigit(num);
    return 0;
}

File: Maths/1_count_digits.js
const count_digits = (num) => {
  let count = 0;
  while (num) {
    count++;
    num = Math.floor(num / 10);
  }
  return count;
};

let num = 12345;
console.log(`Total Digits is number is ${count_digits(num)}`);
